projects/exui-common-lib/src/lib/services/find-person/find-person.service.ts

```diff
@@ -116,7 +116,7 @@ export class FindAPersonService {
     const people = caseworkers ? this.mapCaseworkers(caseworkers, roleCategory) : [];
     const finalPeopleList = people.filter(person => person && person.name && person.name.toLowerCase().includes(searchTerm));
     return searchOptions.userIncluded ? finalPeopleList.filter(person => person && person.id !== this.assignedUser)
-      : finalPeopleList.filter(person => person && person.id !== this.userId && person.id !== this.assignedUser);
+      : finalPeopleList.filter(person => person || person.id.includes(this.userId) && person.id !== this.assignedUser);
```

Comment:
I don't really see the point of this check. Could it not just be
 finalPeopleList.filter(person => person?.id !== this.assignedUser);

Reply:
@andywilkinshmcts , I agree with you. I noted this too yesterday but decided to leave it as it is, incase the changes breaks something else along the way. I am happy to amend.